RELEASES.md

```diff
@@ -1,149 +1,144 @@
-### Alpha - Release Candidate phase (2-4 weeks prior to release)
-The goal of this phase is to prepare a polished and stable release:
-- Remove any “work in progress” code that had been added as part of a larger feature that is not yet complete.
-- Complete a full accessibility review
-- Complete integration testing
-- Document on the website a complete list of all new features and changes. Highlight key new features.
-### Stable Release (each March and September)
-The goal of the stable release is to provide a version of Quickstart for The University of Arizona community to use that will be supported for a full year with security updates and bug fixes while minimizing any other disruptions.
-### End of Life
-Once two more releases of Quickstart are available (approximately one year from initial release), releases will no longer be supported. This means that they will not receive any further security updates or bug fixes.
--->
-
-### Beta Releases (beta)
-Beta releases serve as a preview of the upcoming stable release and are meant for testing purposes. They provide developers and site maintainers an opportunity to test new features, APIs, and compatibility with existing configurations. Feedback from beta testing is crucial to address bugs, performance issues, or any other concerns before the stable release.
-
-### Release Candidates (rc)
-Release candidates are the final step before the stable release. These versions are considered feature complete, with all proposed features implemented and bugs addressed. The primary purpose of an RC is to ensure that there are no critical issues that were missed during beta testing. If no significant problems are identified in an RC, it can be promoted to a stable release. However, if issues are found, they are addressed, and a new RC is issued for testing.
+
+The overarching goal of this release policy is to ensure developers and site maintainers know what to expect when using Arizona Quickstart. Arizona Quickstart follows [semantic versioning](https://semver.org/) and aligns with Drupal's core release cycle to allow predictable updates, especially for security and stability.
+
+## Release Support Policy
+
+Arizona Digital supports two minor releases of Arizona Quickstart at a time. Patch releases are issued as needed. However, we may designate specific minor releases as **Long Term Support (LTS)** versions when a future release introduces significant change. In these cases, the LTS version will continue receiving patch releases alongside a single newer release.
+
+### Example Support Scenario
+
+To illustrate how the LTS model works, consider the following example:
+
+- **2.13** is designated as a **Long Term Support (LTS)** release.
+- **2.14** is a regular minor release that includes more disruptive changes.
+- When **2.15** is released, **2.13 (LTS)** and **2.15** will be supported concurrently.
+- Support for **2.14** ends at that point.
```

Comment:
I think I was imagining something different: when 2.15 is released, both 2.14. and 2.13. would continue to be supported but 2.13 would only be receiving security or critical bug fixes at tht point.

Reply:
Are you sure that we want to support three simultaneous releases?

Reply:
I guess it depends on exactly what we decide we need to include in LTS patch releases.  If it's security updates only or security updates + minimal other critical changes, I think I'd potentially be OK with having 3 overlapping supported releases for some amount of time.

Reply:
We decided 2.14.x will be our Quickstart 2 LTS release and that minor Drupal core releases created between now and its end of life will all be included in Quickstart 2.14.x.